Sources/Asteroids/main.swift

```diff
@@ -0,0 +1,338 @@
+import FirebladeECS
+import Library
+import SDL2
+
+// MARK: SDL and Nexus Engines Setup
+
+// swiftlint:disable prefixed_toplevel_constant
+
+// initialize the SDL library with video and audio
+if SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO) != 0 {
+    fatalError("could not init video/audio")
```

Comment:
It would be nice to show an error reason to the user so the user is able to do something about initialization problems.
I would do it similar to this:

```swift
if SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO) != 0 {
    fatalError("Could not initialize video/audio - reason: \(String(cString:  SDL_GetError()))")
}
```